index.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+require_once 'functions.php';
+
+$db = createDbConn();
+$films = displayDb($db);
+
+$filmHtml = displayFilms($films);
+
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="normalize.css"/>
+    <link rel="stylesheet" type="text/css" href="styles.css"/>
+    <title>Disney Film Collection</title>
+</head>
+
+<body>
+
+<header>
+        <h1> Disney Film Collection</h1>
+</header>
+
+<main>
+    <section class="overall">
+        <div class="film_collection">
+        <?php echo $filmHtml; ?>
+        </div>
+    </section>
+</main>
+
+<section class="button_section">
```

Comment:
will need to remove this button as it is not currently functioning. it can be re added in the next story. this gives us clear boundaries between the work and is good practice

Reply:
Button HMTL and CSS removed